packages/hierarchies-react/src/presentation-hierarchies-react/UseTree.ts

```diff
@@ -116,20 +116,52 @@ interface ReloadTreeOptions {
 }
 
 /** @public */
-export interface UseTreeResult {
+export type UseTreeResult = {
   /**
-   * Array containing root tree nodes. It is `undefined` on initial render until any nodes are loaded.
-   */
-  rootNodes: PresentationTreeNode[] | undefined;
-  /**
-   * Specifies whether tree is loading or not. It is set to `true` when initial tree load is in progress
+   * Specifies whether tree is reloading or not. It is set to `false` when initial tree load is in progress to check if
    * or tree is reloading.
    */
-  isLoading: boolean;
+  isReloading: boolean;
+  /** Get a tree node by id */
+  getNode: (nodeId: string) => PresentationHierarchyNode | undefined;
+  /** Sets a formatter for the primitive values that are displayed in the hierarchy. */
+  setFormatter: (formatter: IPrimitiveValueFormatter | undefined) => void;
   /**
    * A function that should be called to reload the tree.
    */
   reloadTree: (options?: ReloadTreeOptions) => void;
+  /** Returns hierarchy level details for a given node ID. */
+  getHierarchyLevelDetails: (nodeId: string | undefined) => HierarchyLevelDetails | undefined;
+} & RenderProps;
+
+export type RenderProps =
+  | {
+      /**
+       * An object containing information used to render root error handling UI.
+       * Use in combination with `RootErrorRenderer` or your custom implementation.
+       * Defined when root nodes fail to load.
+       */
+      rootErrorRenderProps: {
+        /** Object containing root error information */
+        rootError: ErrorInfo;
```

Comment:
maybe just call this `error`? currently there's a bit of a repetition

Reply:
Changed